testcontainers/Cargo.toml

```diff
@@ -7,7 +7,7 @@ edition = "2021"
 keywords = [ "docker", "testcontainers" ]
 license = "MIT OR Apache-2.0"
 repository = "https://github.com/testcontainers/testcontainers-rs"
-rust-version = "1.60.0"
+rust-version = "1.70.0"
```

Comment:
I am using sync::OnceLock which was added in 1.70 to store static value  but if we want to still be in 1.60 we can achieve the same with using something like https://crates.io/crates/once_cell

Reply:
In general I'd expect users to run tests with a recent Rust compiler so this is fine.